src/signals/incident-management/containers/IncidentSplitContainer/IncidentSplitContainer.test.js

```diff
@@ -345,7 +345,7 @@ describe('signals/incident-management/containers/IncidentSplitContainer', () =>
 
     expect(dispatch).toHaveBeenCalledWith(
       showGlobalNotification({
-        title: 'De deelmelding kon niet gemaakt worden',
+        title: 'De deelmelding(en) kon niet gemaakt worden',
```

Comment:
Here too

Reply:
True - although this occurrence will always reflect the text in the main file